ui/admin/app/components/form/host-set/azure/index.hbs

```diff
@@ -51,16 +51,11 @@
     {{/if}}
   </Hds::Form::Textarea::Field>
 
-  <InfoField @value={{@model.type}} readonly={{true}} as |F|>
+  <InfoField @value={{@model.type}} as |F|>
```

Comment:
Can you share more context about this removal?

Reply:
The InfoField component has `readonly` set to true in the component itself and it currently doesn't allow for dynamically setting that attribute. So we didn't need to pass these values in.
A small caveat is the InfoField component has `...attributes` which allows the user to pass anything to it. So in this case we were passing an attribute that already existed. In case of passing `disabled` to the component, it was adding disabled and readonly to the html element which is not ok either.
Please let me know if you have more questions.